sass/styles.scss

```
@@ -0,0 +1,505 @@
+$font-sans:'Open Sans',
+sans-serif;
+$blue:#0240f4;
+$square-cell:minmax(100px, min(300px, 33vw));
+$square-cell2:minmax(100px, min(300px, 16vw));
+
+* {
+    box-sizing: border-box;
+}
+
+//gneral text
+
+body {
+    background-color: darkgray;
+    color: $blue;
+    font-family: $font-sans;
+    font-size: 14px;
+    box-shadow: 30px 50px 10px grey;
+    margin: 0;
+}
+
+div {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+}
+
+h1 {
+    font-size: 160px;
+    line-height: 1.3;
+    margin: 0;
+}
+
+h2 {
+    align-items: baseline;
+    font-size: 160px;
+    line-height: 0.8;
+    margin: 0;
+}
+
+h3 {
+    margin: 0 0 0 5px;
+}
+
+main {
+    background-color: white;
+    display: grid;
+    grid-template-columns: repeat(3, $square-cell);
+    grid-template-rows: repeat(16, $square-cell);
+}
+
+header,
+footer {
+    background-color: white;
+    display: grid;
+    grid-template-columns: repeat(3, $square-cell);
+    grid-template-rows: auto;
+    box-shadow: 30px 50px 10px grey;
+}
```

Same note as above re: combining styles:
```suggestion
main, header, footer {
    background-color: white;
    display: grid;
    grid-template-columns: repeat(3, $square-cell);
}

main {
    grid-template-rows: repeat(16, $square-cell);
}

header,
footer {
    grid-template-rows: auto;
    box-shadow: 30px 50px 10px grey;
}
```